pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go

```diff
@@ -323,6 +360,15 @@ func (dswp *desiredStateOfWorldPopulator) processPodVolumes(
 				format.Pod(pod),
 				err)
 			dswp.desiredStateOfWorld.AddErrorToPod(uniquePodName, err.Error())
+			if strings.Contains(err.Error(), pvcBeingDeletedMsg) {
+				if pvcSource := podVolume.VolumeSource.PersistentVolumeClaim; pvcSource != nil {
+					klog.V(4).Infof("pvc %s/%s is being deleted", pod.Namespace, pvcSource.ClaimName)
+					if dswp.pvcsUnderDeletion == nil {
+						dswp.pvcsUnderDeletion = make(map[string]bool)
+					}
+					dswp.pvcsUnderDeletion[pod.Namespace+"/"+pvcSource.ClaimName] = true
```

Comment:
This key is used multiple times. It may be better have get/set wrappers around it to make sure the keys used are consistent.